core/roboconf-dm/src/main/java/net/roboconf/dm/internal/api/impl/ApplicationMngrImpl.java

```diff
@@ -351,4 +357,25 @@ private ManagedApplication createApplication(
 		this.logger.info( "Application " + name + " was successfully created from the template " + tpl + "." );
 		return ma;
 	}
+
+
+	@Override
+	public void bindApplication( ManagedApplication ma, String applicationTemplateName, String applicationName )
```

Comment:
Yes it is.
In this sitution (inter-application dependencies), the application template name designates the prefix of an imported variable.